src/runAll.js

```diff
@@ -62,51 +62,56 @@ module.exports = function runAll(config) {
       renderer
     }
 
-    if (tasks.length) {
-      // Do not terminate main Listr process on SIGINT
-      process.on('SIGINT', () => {})
+    // If all of the configured "linters" should be skipped
+    // avoid executing any lint-staged logic
+    if (tasks.every(task => task.skip())) {
+      console.log(
+        'No linters executed. No staged files match any of provided globs in "lint-staged.linters".'
```

Comment:
Let’s fix grammar here and it’s good to merge. I’d say drop the `No linters executed.` part. Also drop `in “lint-staged.linters”`

Reply:
Sure. I figured if we were going to report an error message, we'd make it clear precisely what we mean by "provided globs", config key-wise.

Reply:
I've updated the copy :)